modules/db/src/main/scala/gem/dao/StaticConfigDao.scala

```diff
@@ -67,77 +64,172 @@ object StaticConfigDao {
-          FROM static_gmos_south
-         WHERE static_id = $sid AND instrument = ${Instrument.GmosS: Instrument}
-      """.query[StaticConfig.GmosSouth]
-
     def insertBaseSlice(i: Instrument): Update0 =
       sql"""
         INSERT INTO static_config (instrument)
         VALUES ($i)
       """.update
 
-    def insertF2(id: Int, f2: StaticConfig.F2): Update0 =
-      sql"""
-        INSERT INTO static_f2 (static_id, instrument, mos_preimaging)
-        VALUES (
-          $id,
-          ${Instrument.Flamingos2: Instrument},
-          ${f2.mosPreImaging})
-      """.update
+    /** F2 Statements. */
+    object F2 {
+
+      import Instrument.Flamingos2
+
+      def select(sid: Int): Query0[StaticConfig.F2] =
+        sql"""
+          SELECT mos_preimaging
+            FROM static_f2
+           WHERE static_id = $sid AND instrument = ${Flamingos2: Instrument}
+        """.query[StaticConfig.F2]
+
+      def insert(id: Int, f2: StaticConfig.F2): Update0 =
+        sql"""
+          INSERT INTO static_f2 (static_id, instrument, mos_preimaging)
+          VALUES (
+            $id,
+            ${Flamingos2: Instrument},
+            ${f2.mosPreImaging})
+        """.update
+    }
 
-    def insertGmosNorth(id: Int, g: StaticConfig.GmosNorth): Update0 =
-      sql"""
-        INSERT INTO static_gmos_north (static_id, instrument, detector, mos_preimaging, stage_mode)
-        VALUES (
-          $id,
-          ${Instrument.GmosN: Instrument},
-          ${g.common.detector},
-          ${g.common.mosPreImaging},
-          ${g.stageMode})
-      """.update
+    /** GMOS Statements. */
+    object Gmos {
+
+      import gem.config.Gmos.{ GmosCommonStaticConfig => GmosCommonSC, GmosNodAndShuffle, GmosShuffleCycles, GmosShuffleOffset }
+      import gem.enum.Instrument.{ GmosN, GmosS }
+      import StaticConfig.{ GmosNorth, GmosSouth }
+
+      // We need to define this explicitly because we're ignoring the nod and
+      // shuffle bit.
+      implicit val GmosCommonStaticComposite: Composite[GmosCommonSC] =
+        Composite[(GmosDetector, MosPreImaging)].xmap(
+          (t: (GmosDetector, MosPreImaging)) => GmosCommonSC(t._1, t._2, None),
+          (s: GmosCommonSC) => (s.detector, s.mosPreImaging)
+        )
+
+      implicit val MetaGmosShuffleOffset: Meta[GmosShuffleOffset] =
+        Meta[Int].xmap(GmosShuffleOffset.unsafeFromRowCount, _.detectorRows)
+
+      implicit val MetaGmosShuffleCycles: Meta[GmosShuffleCycles] =
+        Meta[Int].xmap(GmosShuffleCycles.unsafeFromCycleCount, _.toInt)
+
+      def selectNodAndShuffle(sid: Int, i: Instrument): Query0[GmosNodAndShuffle] =
+        sql"""
+          SELECT a_offset_p,
+                 a_offset_q,
+                 b_offset_p,
+                 b_offset_q,
+                 e_offset,
+                 offset_rows,
+                 cycles
+            FROM gmos_nod_and_shuffle
+           WHERE static_id = $sid AND instrument = $i
+        """.query[GmosNodAndShuffle]
```

Comment:
I could join `gmos_nod_and_shuffle` when selecting the static config instead.  This seemed simple though.  Opinions?